kythe/go/extractors/validation/validation.go

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package validation contains logic for verifing the contents of a kzip.
```

Comment:
Can you add some more detail here about what it means for a kzip to be "valid"? A link to a separate doc would be fine, but it's nice to have some context when viewing libraries on godoc.org

Reply:
:+1: 

Let me know how much more wording you want here.

Reply:
That seems reasonable. I was just looking for something to give the reader the high-level idea.